src/drawer.js

```diff
@@ -288,6 +288,22 @@ $.Drawer.prototype = /** @lends OpenSeadragon.Drawer.prototype */{
     },
 
     // private
+    drawPlaceholder: function(rect, fillStyle) {
+        if (!this.useCanvas) {
+            return;
+        }
+
+        if ( typeof fillStyle === "function" ) {
+            this.context.fillStyle = fillStyle(this.context);
```

Comment:
I passed the context into the function to be able to draw a gradient or a image like this:

```javascript
placeholderFillStyle: function(context) {
    var gradient = context.createLinearGradient(0, 0, context.canvas.width, 0);
    gradient.addColorStop(0,"black");
    gradient.addColorStop(1,"red");

    return gradient;
}
```